Cannot find module 'NodObjC'

When I try to invoke any isim command (except help), I am seeing the following error message:

Cannot find module 'NodObjC'

I'm using the following:

node 0.12.9
npm 2.14.9
osx 10.11.2
xcode 7.2

--- a/lib/iphone-simulator-xcode-7.ts
+++ b/lib/iphone-simulator-xcode-7.ts
@@ -10,7 +10,7 @@ import { Simctl } from "./simctl";
 import util = require("util");
 import utils = require("./utils");
 import xcode = require("./xcode");
-var $ = require("NodObjC");
+var $ = require("nodobjc");
 var osenv = require("osenv");
 
 export class XCode7Simulator implements ISimulator {

